test/test_json.py

```diff
@@ -0,0 +1,15 @@
+import json
+import os
+
+HOMEDIR = os.path.expanduser('~')
+file_location = os.path.dirname(os.path.realpath(__file__))
+retriever_root_dir = os.path.abspath(os.path.join(file_location, os.pardir))
+
+def json_test_module():
+	"""Test if all json files are valid"""
+	os.chdir(retriever_root_dir + '/scripts')
```

Comment:
Instead of changing the directory, using `os.listdir`, and subsetting based on the last 5 characters of the filename I think using glob will be a little clearer. E.g.,:

```
from glob import glob
...
for filename in glob("./scripts/*.json"):
...
```